xml/System.Collections.Generic/Dictionary`2.xml

```diff
@@ -2715,7 +2715,9 @@ finally {
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Sets the capacity of this dictionary to what it would be if it had been originally initialized with all its entries This method can be used to minimize the memory overhead once it is known that no new elements will be added. To allocate minimum size storage array, execute the following statements:
```

Comment:
Missing a `.` before `This`. 
```suggestion
        <summary>Sets the capacity of this dictionary to what it would be if it had been originally initialized with all its entries. This method can be used to minimize the memory overhead once it is known that no new elements will be added. To allocate minimum size storage array, execute the following statements:
```